Makefile

```diff
@@ -36,7 +40,7 @@ test-e2e: ## Run E2E tests.
 .PHONY: image
 image: export GOOS=linux
 image: build ## Build a Linux based Hegel image for the the host architecture.
-	docker build --build-arg GOPROXY=$(GOPROXY) $(IMAGE_ARGS) .
+	docker build --build-arg TARGETARCH=$(GOARCH) --build-arg TARGETOS=$(GOOS) $(IMAGE_ARGS) .
```

Comment:
I don't think we need the `TARGETARCH` given this is Make target is meant to build for the host architecture.

Reply:
Just trying to get the image to build at all. I found without both of these the image would fail to build.

Reply:
We had a chat offline and clarified some of these built-in variables. It seems [BuildKit is required](https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope) to populate the built-in variables used in our Dockerfile. We don't document this dependency anywhere in Hegel currently.

We should, as a separate issue, figure out whether BuiltKit comes as part of the latest Docker builds and if not chose to either instrument the variables manually or document the requirement.

For now, we should be able to revert these changes and just maintain our hidden BuildKit dependency.